lambdas/db-provision-user-database/src/index.ts

```diff
@@ -0,0 +1,75 @@
+import Knex from 'knex';
+import AWS from 'aws-sdk';
+
+export interface HandlerEvent {
+  rootLoginSecret: string,
+  userLoginSecret: string,
+  prefix: string,
+  dbPassword: string,
+  engine: string,
+  dbClusterIdentifier: string,
+  env?: NodeJS.ProcessEnv,
+}
+
+export interface getSecretValue extends
```

Comment:
What is the reason for this?

Reply:
Cruft from development only.  Removed: 46f6a70f8